build_defs/build_defs.bzl

```diff
@@ -42,14 +43,20 @@ def arcs_cc_schema(name, src, out = None):
 def arcs_kt_schema(name, src, out = None):
     """Generates a Kotlin file for the given .arcs schema file."""
     _run_schema2pkg(
-        name = name,
+        name = name + "_genrule",
```

Comment:
Does this require that we change existing BUILD files that make use of this macro?

Reply:
For example, `particles/Native/Wasm/BUILD`

Reply:
Also `particles/Tutorial/Kotlin/BUILD`